data/datacatalogue/molgenis.csv

```diff
@@ -235,7 +235,7 @@ CollectionEvents,,dataCategories,ontology_array,,,CatalogueOntologies,DataCatego
 CollectionEvents,,sampleCategories,ontology_array,,,CatalogueOntologies,SampleCategories,,,,,Samples that were collected in this collection event
 CollectionEvents,,standardizedTools,ontology_array,,,CatalogueOntologies,StandardizedTools,,,,,"Standardized tools, e.g. surveys, questionnaires, instruments used to collect data for this collection event"
 CollectionEvents,,standardizedToolsOther,,,,,,,,,,"If 'other', please specify"
-CollectionEvents,,coreVariables,ontology_array,,,CatalogueOntologies,CoreVariables,,,,,Select 10-20 relevant variables that were collected in this collection event 
```

Comment:
consider storing the model as json or yml ( instead of csv) to make changes easer to track ( more human readable , as keys and values are combined

Reply:
yes, we certainly want to move towards a yaml, analogous to what we do in FAIR Genomes :-). Out of scope for this requests. BTW this is a simple formatting change.